### PR TITLE
chore(contracts): separate avs into its own forge project

### PR DESCRIPTION
**Pull request by kevinhalliday**
Separate AVS into its own forge project.

This is a prerequisite to upgrading openzeppelin in the core. 

issue: #1310

### CODE REVIEW

**Review by pavelnikolov — APPROVED** [review]

File: contracts/core/script/genesis/AllocPredeploys.s.sol [hunk 1]
```diff
@@ -158,7 +158,7 @@ contract AllocPredeploys is Script {
         bytes32 ownerSlot = bytes32(0);
 
         vm.store(Predeploys.ProxyAdmin, ownerSlot, bytes32(uint256(uint160(cfg.admin))));
-        vm.etch(Predeploys.ProxyAdmin, vm.getDeployedCode("out/ProxyAdmin.sol/ProxyAdmin.0.8.24.json"));
+        vm.etch(Predeploys.ProxyAdmin, vm.getDeployedCode("out/ProxyAdmin.sol/ProxyAdmin.json"));
```
Comment (ttarsi) [inline contracts/core/script/genesis/AllocPredeploys.s.sol:161]: can we confirm that this uses our `ProxyAdmin` and not Open Zeppelines?

**Review by ttarsi — APPROVED** [review]

File: contracts/core/script/genesis/AllocPredeploys.s.sol (hunk 1 above)
Comment (kevinhalliday) [reply contracts/core/script/genesis/AllocPredeploys.s.sol:161]: good call yeah it should

Comment (kevinhalliday) [reply contracts/core/script/genesis/AllocPredeploys.s.sol:161]: confirmed it was - we only have a single ProxyAdmin artifact. But going to update the string we use here to be explicit to ours

Comment (kevinhalliday) [reply contracts/core/script/genesis/AllocPredeploys.s.sol]: though we will change when we upgrade oz - because they added owner to constructor

**Pull request merged**